--- origsrc/nas-1.9.1/config/NetAudio.tmpl	2007-06-16 14:03:20.000000000 -0500
+++ src/nas-1.9.1/config/NetAudio.tmpl	2007-11-29 07:14:36.060625000 -0600
@@ -155,9 +155,15 @@
 
 
 #else /* SharedLibAudio */
+#if defined(cygwinArchitecture)
+ LOCAL_LDFLAGS = _AuUse(-L$(USRLIBDIR), -L$(AUSRC) -L$(USRLIBDIR))
+      DEPAULIB =
+         AULIB = -laudio
+#else /* cygwinArchitecture */
  LOCAL_LDFLAGS =
       DEPAULIB = _AuUseCat($(USRLIBDIR),$(AUSRC),/Lib(audio))
          AULIB = LoaderLibPrefix _AuUse(-laudio,$(DEPAULIB))
+#endif /* cygwinArchitecture */
 #endif /* SharedLibAudio */
 
   AUDIODEPLIBS = $(DEPAULIB) $(DEPXTOOLLIB) $(DEPXLIB)
